src/script.js

```diff
@@ -1,6 +1,5 @@
 import './style.css';
 
-
 const addButton = document.querySelector('.addButton');
 const inputValue = document.querySelector('.input');
 const container = document.querySelector('.container');
```

Comment:
- [ ] Please set the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5).
<img width="1430" alt="Screen Shot 2023-04-07 at 10 29 37" src="https://user-images.githubusercontent.com/44745855/230573383-91997c3c-d068-45cf-a742-2d29953a6215.png">

Reply:
- [ ] I am not able to edit

Reply:
- [ ] By default, new tasks should have the property `completed` set to `false`